Implemented Obj field usage

Fernando-Fernandez opened the pull request:
Implemented new Object Field Usage feature:  this will count how many records has nulls in each field and report the percentage of nulls.
If run with the parameter --field, it will count distinct values in the field (up to 1000) and report the percentages of each value.
The goal is to help determine if a field is being used or not and whether some values dominate (possibly indicating that users are either entering dummy data or perhaps that the field is not that useful).

https://github.com/hardisgroupcom/sfdx-hardis/issues/1517

nvuillam commented on src/commands/hardis/doc/object-field-usage.ts:
```diff
@@ -0,0 +1,393 @@
+
+  public static flags: any = {
+    'target-org': requiredOrgFlagWithDeprecations,
+    sObject: Flags.string({
```
plz can u use 's-objects' or 'objects' ? :)
It would be nice to be able to call the command for multiple objects in one call

In case the option is not sent, you can also prompt the user to select objets that he/she wants to analyze :)

See example of objects selection here: https://github.com/hardisgroupcom/sfdx-hardis/blob/e346bfcd7444c6730950d8af9e2313b0604e6f05/src/commands/hardis/org/diagnose/storage-stats.ts#L202

nvuillam reviewed (changes requested):
Looks amazing @Fernando-Fernandez , thanks :)

Please can you handle the remarks I added in the review ? :)

Also, if before running the queries, it would be possible to estimate the number of API calls that will be consumed and ask for user confirmation ?

Fernando-Fernandez commented:
@nvuillam I believe I implemented all the requested improvements.

- Changed the flag from --sObject to --Objects. 
- It now works for multiple objects in one call.
- Before running the queries, the estimated number of API calls that will be consumed is shown and it prompts the user for confirmation.
- It generates one CSV per object and a summary CSV then aggregates all of them in a single XLS file.
- Renamed the --field parameter to --fields and made it accept multiple fields.
- When no field is specified, it will run it for all relevant fields.
- When --objects is not specified, it will prompt the user to select objects.
- When --objects is specified with no value, like above, it will prompt the user to select objects.

Fernando-Fernandez commented:
> @Fernando-Fernandez just tested it, it seems to work well :)
> 
> last update; in th XSLX, please can you put the summary tab in the first position ?

Done. ✅

Fernando-Fernandez commented:
> Wonderful PR @Fernando-Fernandez :) i'll release it in beta until the next minor version :)

Awesome! Great ideas, by the way!
